server/entities/User.ts

```diff
@@ -25,10 +25,27 @@ export class User {
     @Column('varchar', { nullable: true })
     familyName: string | null;
 
+    @Column('varchar', { nullable: true })
+    biography: string | null;
+
     @Column('int', { length: 2 })
     role: UserRole = UserRole.Student;
 
     get roleName() {
         return UserRole[this.role];
     }
+
+    get url() {
+        return '/profile/' + this.id
+    }
+
+    set bio(bio: string) {
+        if(bio && bio.length >= 250) {
+            this.bio = bio;
```

Comment:
I think this is infinitely recursive.

I think it would be better to put this kind of validation logic in a method like `updateFromData` which takes the entire req.body object and updates everything, not just the bio